src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -64,10 +64,14 @@ class DeprecationErrorHandler
      * The mode can alternatively be "/some-regexp/" to stop the test suite whenever
      * a deprecation message matches the given regular expression.
      *
-     * @param int|string|false $mode The reporting mode, defaults to not allowing any deprecations
+     * @param ?int|string|false $mode The reporting mode, defaults to not allowing any deprecations
```

Comment:
I'm not sure we're using this style anywhere in current docblock, `|null` would be more clear I think.